shell/common/engine.h

```diff
@@ -677,16 +677,16 @@ class Engine final : public RuntimeDelegate, PointerDataDispatcher::Delegate {
   std::optional<uint32_t> GetUIIsolateReturnCode();
 
   //----------------------------------------------------------------------------
-  /// @brief      Updates the viewport metrics for the currently running Flutter
-  ///             application. The viewport metrics detail the size of the
-  ///             rendering viewport in texels as well as edge insets if
-  ///             present.
+  /// @brief      Updates the viewport metrics for a view. The viewport metrics
+  ///             detail the size of the rendering viewport in texels as well as
```

Comment:
Is texels correct here? It looks like that's [a thing](https://en.wikipedia.org/wiki/Texel_(graphics)) but it seems like pixels would make more sense.

Reply:
This comment was written by @/chinmaygarde. Maybe you can ask him for details? (Since I'm not really familiar with texels either...)